python/setup.py

```diff
@@ -22,6 +22,7 @@
     "ethereum": ["rlp>=1.1.0", "web3>=4.8"],
     "qt-widgets": ["PyQt5"],
     "extra": ["Pillow"],
+    "stellar": ["stellar-sdk>=4.0.0,<5.0.0"],
```

Comment:
Do I need to add it to pyproject.toml in the root directory? If necessary, is it to be added under `cointool`?